spot_wrapper/wrapper.py

```diff
@@ -990,104 +728,127 @@ def authenticate(robot, username, password, logger):
         return authenticated
 
     @property
-    def robot_name(self):
+    def robot_name(self) -> str:
         return self._robot_name
 
     @property
-    def frame_prefix(self):
+    def frame_prefix(self) -> str:
         return self._frame_prefix
 
     @property
-    def logger(self):
+    def spot_images(self) -> SpotImages:
+        """Return SpotImages instance"""
+        return self._spot_images
+
+    @property
+    def spot_arm(self) -> SpotArm:
+        """Return SpotArm instance"""
+        if not self._robot.has_arm():
+            raise Exception("SpotArm is not available on this robot")
```

Comment:
Please make a custom exception class called "MissingSpotArm" or the like.

Reply:
Added in 0ec85565f653680fdcdf1dcfd65dd93803faedda